File: Python/Human.py
# Saya Hakasa Putri mengerjakan LP3 DPBO 2023 C2 
# dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
# kecurangan seperti yang telah dispesifikasikan. Aamiin 


class Human:


    __NIK = ""
    __nama = ""
    __gender = ""

    def __init__(self): #Constructor

        # atribut
        self.__NIK = ""
        self.__nama = ""
        self.__gender = ""

    # metode getter dam setter

    def getNIK(self):
        return self.NIK
    
    def setNIK(self, NIK):
        self.NIK = NIK
    
    def getNama(self):
        return self.nama
    
    def setNama(self, nama):
        self.nama = nama

    def getGender(self):
        return self.Gender
    
    def setGender(self, Gender):
        self.Gender = Gender

    # untuk print atribut kelas human
    def getInfoHuman(self):

        print("NIK            : " + self.getNIK())
        print("Nama           : " + self.getNama())
        print("Gender         : " + self.getGender())

    

File: C++/Program/Human.cpp
// Saya Hakasa Putri mengerjakan LP3 DPBO 2023 C2
// dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan
// kecurangan seperti yang telah dispesifikasikan. Aamiin

#include <iostream>
#include <string>

using namespace std;

class Human {
private:
string __NIK;
string __nama;
string __gender;

public:
Human() { // Constructor
__NIK = "";
__nama = "";
__gender = "";
}

// Metode getter dan setter
string getNIK() {
    return __NIK;
}

void setNIK(string NIK) {
    __NIK = NIK;
}

string getNama() {
    return __nama;
}

void setNama(string nama) {
    __nama = nama;
}

string getGender() {
    return __gender;
}

void setGender(string Gender) {
    __gender = Gender;
}

// untuk print atribut kelas human
void getInfoHuman() {
    cout << "NIK            : " << getNIK() << endl;
    cout << "Nama           : " << getNama() << endl;
    cout << "Gender         : " << getGender() << endl;
}

 ~Human(){ //destruktor
    }

};
